packages/contracts/contracts/libs/v0.8.x/minter-libs/SetPriceLib.sol

```diff
@@ -0,0 +1,49 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+// Created By: Art Blocks Inc.
+
+// import "../../../interfaces/v0.8.x/IMinterBaseV0.sol";
+// import "../../../interfaces/v0.8.x/IGenArt721CoreContractV3_Base.sol";
+// import "../../../interfaces/v0.8.x/IGenArt721CoreContractV3.sol";
+// import "../../../interfaces/v0.8.x/IGenArt721CoreContractV3_Engine.sol";
```

Comment:
nit: can these be removed since unused/commented out?

Reply:
oops! nice catch - commented these out to check that they were unused imports, then forgot to remove! updated!